### PR TITLE
assigning right error oode when partial snapshot failure

### PR DESCRIPTION
**Pull request by bpramod**

### CODE REVIEW

File: VMBackup/main/freezesnapshotter.py [hunk 1]
```diff
@@ -159,8 +159,8 @@ def updateErrorCode(self, blob_snapshot_info_array, all_failed):
                         error_msg = error_msg + ExtensionErrorCodeHelper.ExtensionErrorCodeHelper.StatusCodeStringBuilder(self.extensionErrorCode)
                         self.logger.log(error_msg, True, 'Error')
                         break
-                    elif blob_snapshot_info.isSuccessful == False and not all_failed:
```
Comment (vityagi) [inline VMBackup/main/freezesnapshotter.py]: this elif should still be kept

Comment (bpramod) [reply VMBackup/main/freezesnapshotter.py]: Yes, i think i should still keep that. will do that change.

**Comment by vityagi** [comment]
Looks fine.

**Comment by vityagi** [comment]
Looks good.

**Pull request merged**